lib/matplotlib/figure.py

```diff
@@ -1067,7 +1067,7 @@ def draw(self, renderer):
             im.is_grayscale = False
             l, b, w, h = self.bbox.bounds
 
-            def draw_composite():
+            def draw_combined_image():
```

Comment:
another API change. Wouldn't this impact third-party backends like IPython's internal backend?